### PR TITLE
Changes from Western New England Comp

### PR DESCRIPTION
**Pull request by AdamCraftmaster**

### CODE REVIEW

File: src/main/java/frc/robot/commands/auto/complex/BackUpAndBalance.java [hunk 1]
```diff
@@ -13,8 +13,8 @@ public BackUpAndBalance(String relativePosition, Drive drive, Arm arm) {
     int aprilTag = 7;
     addCommands(
         new Initialize(aprilTag, relativePosition, drive, arm),
-        new StraightDriveToPose(Units.inchesToMeters(170.0), 0.0, 0.0, drive),
-        new StraightDriveToPose(Units.inchesToMeters(-75.0), 0.0, 0.0, drive),
+        new StraightDriveToPose(Units.inchesToMeters(175.0), 0.0, 0.0, drive),
```
Comment (elleryjh) [inline src/main/java/frc/robot/commands/auto/complex/BackUpAndBalance.java:16]: Can all these constants be moved to a `FieldConstants` file to give them names and be able to reuse them?

File: src/main/java/frc/robot/subsystems/intakerelease/IntakeRelease.java [hunk 2]
```diff
@@ -60,12 +60,13 @@ public void setWants(Wants wants) {
...
     intakeReleaseIO.updateInputs(inputs, wants);
     logger.processInputs("IntakeRelease", inputs);
+    logger.recordOutput("IntakeRelease/State", state.toString());
 
     switch (state) {
       case DISABLED -> intakeReleaseIO.setPercent(0);
       case INTAKE -> intakeReleaseIO.setPercent(-1.0);
       case RELEASE -> intakeReleaseIO.setPercent(0.6);
-      case HOLD_CUBE -> intakeReleaseIO.setPercent(-0.3);
+      case HOLD_CUBE -> intakeReleaseIO.setPercent(0.0); // -0.3
```
Comment (elleryjh) [inline src/main/java/frc/robot/subsystems/intakerelease/IntakeRelease.java:69]: Why is this 0?

File: src/main/java/frc/robot/commands/auto/complex/BackUpAndBalance.java (hunk 1 above)
Comment (AdamCraftmaster) [reply src/main/java/frc/robot/commands/auto/complex/BackUpAndBalance.java:16]: the goal is to replace these with different positions already in fieldconstants, but this requires us to have more reliable pose estimation first

File: src/main/java/frc/robot/subsystems/intakerelease/IntakeRelease.java (hunk 2 above)
Comment (AdamCraftmaster) [reply src/main/java/frc/robot/subsystems/intakerelease/IntakeRelease.java:69]: we do not want to intake while holding a cube, as it can pop

**Review by elleryjh — APPROVED** [review]

File: src/main/java/frc/robot/commands/intakerelease/HoldCMD.java [hunk 3]
```diff
@@ -29,6 +43,6 @@ public void execute() {
   public void end(boolean interrupted) {}
 
   public boolean isFinished() {
-    return intakerelease.wantsCone() || intakerelease.wantsCube();
+    return false;
```
Comment (john-petrangelo) [inline src/main/java/frc/robot/commands/intakerelease/HoldCMD.java:46]: This is the default behavior, should we just remove isFinished()?

File: build.gradle
Comment (john-petrangelo) [inline build.gradle:1]: Do we need this empty file?

File: src/main/java/frc/robot/subsystems/led/Led2023.java [hunk 4]
```diff
@@ -24,7 +24,7 @@
 public class Led2023 extends SubsystemBase {
   public DoubleLEDStrip ledStrip;
   private final boolean USE_BATTERY_CHECK = true;
-  private final double BATTER_MIN_VOLTAGE = 10.0;
+  private final double BATTER_MIN_VOLTAGE = 9.0; // 10.0
```
Comment (john-petrangelo) [inline src/main/java/frc/robot/subsystems/led/Led2023.java:27]: Remove the comment.

**Review by john-petrangelo — COMMENTED** [review]
lgtm

**Pull request merged**